daemon/algod/api/algod.oas2.json

```diff
@@ -4092,6 +4096,37 @@
         }
       }
     },
+    "AppStateOperation": {
+      "description": "An operation against an app global/local/box key-value pair.",
+      "required": [
+        "operation-type",
+        "app-state-type",
+        "app-id",
+        "key"
+      ],
+      "properties": {
+        "operation-type": {
```

Comment:
changed to `operation` and `application-state`, rather than `application-state-type`.

Reply:
On app-state, I think the "type" is a useful suffix, because without type, the name "app-state" makes you think the value is going to be actual application state, rather than just which kind of application state is being described.

Also, I'm ok with "app" in json key names, I just want us to say "application" when we explain objects in the "description" fields.